docs/sources/trackers/google-tag-manager/snowplow-template/tag-types/index.md

```diff
@@ -0,0 +1,327 @@
+
+
+<details plain>
+<summary>Enhanced Consent Parameters</summary>
+
+| Attribute            | Description                                                                         | Example                 |
+| -------------------- | ----------------------------------------------------------------------------------- | ----------------------- |
+| `basisForProcessing` | GDPR lawful basis for data collection & processing.                                 | Consent                 |
+| `consentUrl`         | URI of the privacy policy related document.                                         | https://example.com     |
+| `consentVersion`     | Version of the privacy policy related document.                                     | 1.0                     |
+| `consentScopes`      | The scopes allowed after the user finalized their selection of consent preferences. | 'analytics, functional' |
+| `domainsApplied`     | The domains for which this consent allows these preferences to persist to.          | https://example.com     |
+| `gdprApplies`        | Determine if GDPR applies based on the user's geo-location.                         | -                       |
+
+</details>
+
+
+### Enhanced Ecommerce
+
+The Enhanced Ecommerce tag is used to track ecommerce events, such as product views, add to cart, and purchases.
+
+See the [Enhanced Ecommerce Tag](/docs/sources/trackers/google-tag-manager/ecommerce-tag-template/index.md) template for more information.
+
+
+### Error Tracking
+
+The Errors tracker tag provides a way to manually track any errors that occur on your website.
+
+<details plain>
+<summary>Error Tracking Parameters</summary>
+
+| **Name**   | **Required?** | **Description**                     | **Example**              |
+| ---------- | ------------- | ----------------------------------- | ------------------------ |
+| `message`  | Yes           | Error message                       | 'Cannot get user object' |
+| `filename` | No            | Filename or URL                     | 'shop.js'                |
+| `lineno`   | No            | Line number of problem code chunk   | 23                       |
+| `colno`    | No            | Column number of problem code chunk | 45                       |
+| `error`    | No            | JS `ErrorEvent`                     | {Stack trace string}     |
+
+</details>
+
+### Page View
+
+You can provide a Custom Page Title if you wish, and you can add a [custom context](/docs/sources/trackers/snowplow-tracker-protocol/custom-events/context-entities/index.md) to the request, as also described [here](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/page-views/index.md#tracking-a-page-view).
+
+If you enable Page Activity Tracking, page pings will be sent at intervals that you can specify.
+
+The Callback Function is something you can set to a JavaScript function. If you set the callback, then instead of sending the page ping to Snowplow, the function gets invoked instead.
```

Comment:
what's with the random capitalisation? are these parameters? should be in a table if so like for the other sections on this page. Or at least marked up as code

is it definitely that the callback is instead of sending the ping, not as well?

Reply:
They're names of the fields in the UI, I'll format them as code.

And yeah, it's [this](https://docs.snowplow.io/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/activity-page-pings/#activity-tracking-callback) in the JS tracker, replaces the pings. I'll link it too.